custom-vu/src/main/kotlin/jces1209/vu/page/AttachScreenShot.kt

```diff
@@ -0,0 +1,26 @@
+package jces1209.vu.page
+
+import jces1209.vu.wait
+import org.openqa.selenium.*
+import org.openqa.selenium.interactions.Actions
+import org.openqa.selenium.support.ui.ExpectedConditions
+
+abstract class AttachScreenShot(
+    protected val driver: WebDriver
+) {
+
+    fun makeScreenShot() {
+        val src = (driver as TakesScreenshot).getScreenshotAs(OutputType.BYTES)
```

Comment:
variable is not used
```suggestion
        (driver as TakesScreenshot).getScreenshotAs(OutputType.BYTES)
```